bindings/const_generator.py

```diff
@@ -56,52 +56,52 @@ def CamelCase(s):
                     'post': '\n',
                 },
                 {   'regex': r'MODE_.*',
-                    'pre': 'bitflags! {\n\tpub flags Mode : u32 {\n',
+                    'pre': 'bitflags! {{\n\tpub flags Mode : u32 {{\n',
```

Comment:
just remove 2nd curly brace, the format would then expect something like {0} and would complain that there's { without }.